.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -316,6 +316,9 @@ steps:
           set -euo pipefail
           cd auditbeat
           mage package
+        retry:
```

Comment:
One problem with packaging failures (which we discovered in the DRA pipeline) is that they may get stuck rather than fail.
My suggestion is that we do the same we did in the [packaging pipeline](https://github.com/elastic/beats/blob/main/.buildkite/packaging.pipeline.yml#L218) i.e. add

```
timeout_in_minutes: 20
```

above the retry statement for all the jobs. From what I've seen so far 20mins should be more than sufficient, typically they take ~7min max, otherwise, get stuck.